gh/templates/gp.sh

```diff
@@ -14,13 +14,14 @@
 
 # usage gp Polymer core-item [branch]
 # Run in a clean directory passing in a GitHub org and repo name
-org=$1
-repo=$2
-branch=${3:-"master"} # default to master when branch isn't specified
+hostname=$1
+org=$2
+repo=$3
+branch=${4:-"master"} # default to master when branch isn't specified
 
 # make folder (same as input, no checking!)
 mkdir $repo
-git clone git@github.com:$org/$repo.git --single-branch
+git clone git@$hostname:$org/$repo.git --single-branch
```

Comment:
The variable has a default value of 'github.com', which when put into that
syntax still works, I believe. I'll double check though.

If a default is provided, does yeoman allow the user to provide a null
value, do you know?
On Dec 8, 2015 4:22 PM, "Rob Dodson" <notifications@github.com> wrote:

> In gh/templates/gp.sh
> <https://github.com/yeoman/generator-polymer/pull/252#discussion_r47015375>
> :
>
> >
> >  # make folder (same as input, no checking!)
> >  mkdir $repo
> > -git clone git@github.com:$org/$repo.git --single-branch
> > +git clone git@$hostname:$org/$repo.git --single-branch
>
> if the user didn't provide a hostname won't this fail?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/yeoman/generator-polymer/pull/252/files#r47015375>.
>